Alignment/OfflineValidation/plugins/TrackerGeometryCompare.cc

```diff
@@ -53,6 +53,8 @@
 #include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
 #include "Geometry/Records/interface/TrackerTopologyRcd.h"
 
+#include "DetectorDescription/Core/interface/DDCompactView.h"
```

Comment:
Can you please explain this file? It is not migrated. Is it part of `GeometricTimingDetExtras`?

Reply:
@cvuosalo in the case of this file, as well as the ```RecoTracker/TkDetLayers/src/Phase2EndcapRingBuilder.cc``` it is a matter of nested include dependencies, that now are exposed by the removal of the ```DDExpandedView``` from ```GeometricDet```. The compilation fails as soon as you update that class.

Reply:
My confusion results from two different versions of this file being included in #29905 and this PR. #29905 has the migrated version. If #29905 is merged first, this PR will probably need to be rebased.

Reply:
@cvuosalo up to the release manager decide which PR is mature for integration first. In case I will rebase mine on that one, not a big issue. BTW, I see that @ianna has already made this addition in #29905